charts/jaeger/README.md

```diff
@@ -1,440 +1,112 @@
+    - **Spark**: Configuration for Spark dependencies job now requires manual environment variable setup via `spark.extraEnv`. The automatic connection logic has been removed.
+- **Component Removal**: The **HotROD** example application has been removed from the chart.
+- **Service Consolidation**: A single Service now exposes all ports (agent, collector, query).
 
-## Deployment Architecture
+## Architecture
 
-### All-In-One (Supported)
-This chart currently uses the **All-In-One** deployment architecture.
-*   **Single Deployable:** All Jaeger components (Collector, Query, Ingester) run within a single application (the compiled Jaeger v2 binary).
-*   **Scalability:** You can scale this Deployment to multiple replicas for high availability, but each replica performs all roles.
-*   **Storage:** While "All-In-One" implies simplicity, **it DOES support persistent storage** (Elasticsearch, Opensearch, Cassandra). You can use this mode for production if your load allows for a unified deployment shape.
-
-### Distributed / Microservices (Not Yet Supported)
-*   **Separate Collectors/Query:** Splitting the stack into separate microservices (e.g., a Deployment for Collectors and a separate Deployment for Query services) is **not currently supported** by this chart configuration.
-
-If you require a fully distributed architecture, you would need to manually create separate values files or fork this chart to create distinct Deployments for each role, utilizing the same Jaeger v2 binary but with different configurations (pipelines).
+This chart uses the **All-In-One** deployment model.
+- **Single Binary**: Runs as a `Deployment` scalable to multiple replicas.
+- **Stateless**: Can connect to external persistent storage (Elasticsearch, Cassandra) for production use.
+- **Default**: Memory storage (ephemeral), suitable for testing.
 
 ## Installing the Chart
 
 Add the Jaeger Tracing Helm repository:
-
 ```bash
 helm repo add jaegertracing https://jaegertracing.github.io/helm-charts
 ```
 
 To install a release named `jaeger`:
-
 ```bash
 helm install jaeger jaegertracing/jaeger
 ```
 
-By default, the chart deploys:
-
-- Jaeger All-in-One Deployment (combines collector, query, and internal components)
-- In-memory storage (non-persistent)
-
-### Overriding the Jaeger Version
-
-The chart version maps to a specific Jaeger version (appVersion). You can override this to use a different Jaeger image version without upgrading the chart:
+## Configuration
 
+### 1. In-Memory (Default)
+Ideal for testing. No persistence.
 ```bash
-helm install jaeger jaegertracing/jaeger \
-  --set allInOne.image.tag=2.13.0
+helm install jaeger jaegertracing/jaeger
 ```
 
-Or via a values file:
+### 2. Elasticsearch (Production Recommended)
+Configure Jaeger to connect to an existing Elasticsearch cluster using `extraEnv`.
 
+**values.yaml Example:**
 ```yaml
 allInOne:
-  image:
-    tag: "2.13.0"
+  extraEnv:
+    - name: SPAN_STORAGE_TYPE
```

Comment:
I'm confused - how do these env vars get applied? jaeger-v1 used to understand them via viper, but with OTEL config they are no longer respected